src/Business Foundation/App/app.json

```diff
@@ -0,0 +1,56 @@
+{
+    "id":  "f3552374-a1f2-4356-848e-196002525837",
+    "name":  "Business Foundation",
+    "publisher":  "Microsoft",
+    "brief":  "Provides a standard set of capabilities that serve as a foundation for developing business apps.",
+    "description":  "Contains an expansive set of open source modules that make it easier to build, maintain, and easily upgrade on-premises and online apps. These modules let you focus on the business logic, and the needs of your users or customers.",
+    "version":  "24.0.0.0",
+    "privacyStatement":  "https://go.microsoft.com/fwlink/?linkid=724009",
+    "EULA":  "https://go.microsoft.com/fwlink/?linkid=2009120",
+    "help":  "https://go.microsoft.com/fwlink/?linkid=2103698",
+    "url":  "https://go.microsoft.com/fwlink/?linkid=724011",
+    "logo": "ExtensionLogo.png",
+    "dependencies":  [
+                        {
+                            "id":  "63ca2fa4-4f03-4f2b-a480-172fef340d3f",
+                            "name":  "System Application",
+                            "publisher":  "Microsoft",
+                            "version":  "24.0.0.0"
+                        }
+    ],
+    "internalsVisibleTo":  [
+                               {
+                                   "id":  "bee8cf2f-494a-42f4-aabd-650e87934d39",
+                                   "name":  "Business Foundation Test Libraries",
+                                   "publisher":  "Microsoft"
+                               }
+                           ],
+    "screenshots":  [
+
+                    ],
+    "platform":  "24.0.0.0",
+    "target":  "OnPrem",
+    "idRanges":  [
+                     {
+                         "from":  1,
+                         "to":  9999
+                     },
+                     {
+                        "from": 12145,
```

Comment:
I'm not sure if I noticed where this range was used. Are they?

Reply:
NoSeriesLineSales.Table and the page, same for purchase. This is existing IT functionality and would be a breaking change to modify.